### DISCUSSION TITLE
Can't run npx gql.tada

### DISCUSSION THREAD

**Discussion by Nielss01** [0no-co/gql.tada#321] [Q&A] [answered]
Hey guys, I have a problem when I try to run npx gql.tada. It doesn't give me an error, but it prompts to be opened. When I click on "Open with Notepad," I paste this script inside:

```
#!/bin/sh
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")

case `uname` in
    *CYGWIN*) basedir=`cygpath -w "$basedir"`;;
esac

if [ -x "$basedir/node" ]; then
  "$basedir/node"  "$basedir/../../../../node_modules/gql.tada/bin/cli.js" "$@"
  ret=$?
else 
  node  "$basedir/../../../../node_modules/gql.tada/bin/cli.js" "$@"
  ret=$?
fi
exit $ret
```
<img width="331" alt="image" src="https://github.com/0no-co/gql.tada/assets/94012670/5719003c-1e26-4310-a1a4-12eb22583dc0">

**Comment by JoviDeCroock** [comment, marked as answer]
On windows you can do `npx gql-tada` instead

**Reply by Nielss01** [reply]
Then i have this error:
npx gql-tada doctor
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module '...\node_modules\node_modules\gql.tada\bin\cli.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

**Reply by JoviDeCroock** [reply]
That does look like a weird error where it tries to access `node_modules` twice, I don't think this is something we can change 😅 because in our case that def exists https://www.runpkg.com/?gql.tada@1.7.6/bin/cli.js maybe you are using an old version or you have install by default turned off.

**Reply by Nielss01** [reply]
It's fixed now. I still can't use `npx` with `gql.tada`, but after installing it globally, I can use `gql-tada`. However, that's another issue with Node itself. The problem was that something went wrong with the installation of `graph.tada` because there were no `node_modules` inside the package, but I got it working. Thanks for the help!
